go/chat/storage/outbox.go

```diff
@@ -31,6 +31,14 @@ type diskOutbox struct {
 	Records []chat1.OutboxRecord `codec:"O"`
 }
 
+func NewOutboxID() (chat1.OutboxID, error) {
```

Comment:
Effectively, I thought the abstraction would be worth it though if we ever want to change it.